includes/wikia/models/DesignSystemFooterModel.class.php

```diff
@@ -0,0 +1,429 @@
+<?php
+
+class DesignSystemFooterModel extends WikiaModel {
```

Comment:
Sure, we can do that. I went with just Footer to keep it consistent with the API method it's providing data for (`getFooter`).